src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -43,7 +43,7 @@ websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selec
 websocket.manual_send.menu                      = Manual Send WebSocket Message
 websocket.manual_send.menu.mnemonic				= w
 websocket.manual_send.popup                     = Resend WebSocket Message
```

Comment:
I knew you were going to ask that.

I guess we could make it closer, but since this makes a 3rd invocation that could be up at the same time it might be best to have a Websocket differentiator? How about `WebSocket Request Editor`?